<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
    <class>GenreFragment</class>
    <widget class="QFrame" name="GenreFragment">
        <property name="geometry">
            <rect>
                <x>0</x>
                <y>0</y>
                <width>1500</width>
                <height>750</height>
            </rect>
        </property>
        <property name="windowTitle">
            <string>Frame</string>
        </property>
        <widget class="QScrollArea" name="scrollArea">
            <property name="geometry">
                <rect>
                    <x>350</x>
                    <y>360</y>
                    <width>801</width>
                    <height>371</height>
                </rect>
            </property>
            <property name="widgetResizable">
                <bool>true</bool>
            </property>
            <widget class="QWidget" name="scrollAreaWidgetContents">
                <property name="geometry">
                    <rect>
                        <x>0</x>
                        <y>0</y>
                        <width>799</width>
                        <height>369</height>
                    </rect>
                </property>
            </widget>
        </widget>
        <widget class="QPushButton" name="pb_changeName">
            <property name="geometry">
                <rect>
                    <x>750</x>
                    <y>260</y>
                    <width>131</width>
                    <height>41</height>
                </rect>
            </property>
            <property name="text">
                <string>Сохранить название</string>
            </property>
        </widget>
        <widget class="QLabel" name="l_Cover">
            <property name="geometry">
                <rect>
                    <x>510</x>
                    <y>120</y>
                    <width>200</width>
                    <height>200</height>
                </rect>
            </property>
            <property name="text">
                <string/>
            </property>
        </widget>
        <widget class="QLineEdit" name="LE_name">
            <property name="geometry">
                <rect>
                    <x>750</x>
                    <y>190</y>
                    <width>371</width>
                    <height>61</height>
                </rect>
            </property>
            <property name="font">
                <font>
                    <family>Segoe UI Black</family>
                    <pointsize>12</pointsize>
                    <weight>75</weight>
                    <bold>true</bold>
                </font>
            </property>
            <property name="styleSheet">
                <string notr="true">border: none</string>
            </property>
        </widget>
        <widget class="QPushButton" name="pb_changeCover">
            <property name="geometry">
                <rect>
                    <x>580</x>
                    <y>60</y>
                    <width>131</width>
                    <height>41</height>
                </rect>
            </property>
            <property name="text">
                <string>Изменить обложку</string>
            </property>
        </widget>
        <widget class="QLabel" name="label">
            <property name="geometry">
                <rect>
                    <x>750</x>
                    <y>130</y>
                    <width>371</width>
                    <height>51</height>
                </rect>
            </property>
            <property name="font">
                <font>
                    <family>Segoe UI</family>
                    <pointsize>12</pointsize>
                    <weight>75</weight>
                    <bold>true</bold>
                </font>
            </property>
            <property name="text">
                <string>Жанр</string>
            </property>
        </widget>
    </widget>
    <resources/>
    <connections/>
</ui>
